### ISSUE TITLE
[JS]Topic: Iterables, Sets and Maps in JavaScript Objects 

### ISSUE THREAD

**Issue by daminimehra** [girlscript/winter-of-contributing#1833]
### Description

 # Iterables, Sets and Maps in JavaScript Objects
In this, We'll discuss Iterables, Sets, and Maps in JavaScript Objects

## Directory
Topics/5.Syntax/Objects/

# Description
I want to update the already made README file for Javascript  objects with some additional contents

### Domain

JavaScript

### Type of Contribution

Documentation

### Code of Conduct

- [X] I follow [Contributing Guidelines](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CONTRIBUTING.md) & [Code of conduct](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CODE_OF_CONDUCT.md) of this project.

**Comment by daminimehra** [comment]
@paramsiddharth @s4-2 
I would like to work on this issue pls assign it to me.

**Comment by SakshiMishra1** [comment]
I would like to work on this issue.

**Comment by daminimehra** [comment]
@SakshiMishra1 ?

**Comment by akratikushwah** [comment]
Hey! I want to work on this issue

**Comment by SakshiMishra1** [comment]
@daminimehra yes?

**Comment by akratikushwah** [comment]
Please assign it to me

**Comment by s4-2** [comment]
Hey @akratikushwah, its already assigned to @daminimehra. You can look for some other issue there are a lot issue under the tag of JavaScript.

**Comment by daminimehra** [comment]
> Hey @akratikushwah, its already assigned to @saksham0309 . You can look for some other issue there are a lot issue under the tag of JavaScript.

@s4-2  mam, why was not assigned to me.?

**Comment by paramsiddharth** [comment]
@daminimehra @akratikushwah It was the automation misbehaving, I apologize for that. You are assigned now. I am going to make some changes in the issue description, please monitor.

**Comment by SakshiMishra1** [comment]
@paramsiddharth sir I have worked on this issue as I was assigned to it. Please review.

**Comment by paramsiddharth** [comment]
> @paramsiddharth sir I have worked on this issue as I was assigned to it. Please review.

@SakshiMishra1 I couldn't find your PR, Sakshi. Are you sure you mentioned the issue number in the PR description?

**Comment by SakshiMishra1** [comment]
> > @paramsiddharth sir I have worked on this issue as I was assigned to it. Please review.
> 
> @SakshiMishra1 I couldn't find your PR, Sakshi. Are you sure you mentioned the issue number in the PR description?

Yes sir, above the comments here, there is a comment that I have referenced the issue.

**Comment by akratikushwah** [comment]
> @daminimehra @akratikushwah It was the automation misbehaving, I apologize for that. You are assigned now. I am going to make some changes in the issue description, please monitor.

Okay

**Comment by paramsiddharth** [comment]
@SakshiMishra1 Please mention the PR number here.

**Comment by SakshiMishra1** [comment]
> @SakshiMishra1 Please mention the PR number here.

@paramsiddharth #1881

**Comment by daminimehra** [comment]
Please review my PR #1998
@paramsiddharth @s4-2

**Comment by daminimehra** [comment]
pls review my PR #2119
@paramsiddharth @s4-2

**Comment by SakshiMishra1** [comment]
Sir please review my PR #2128
@paramsiddharth

**Comment by SakshiMishra1** [comment]
Updated @paramsiddharth PR #2128

**Comment by akratikushwah** [comment]
I would not be able to work on this issue please unassign it to me.
